src/hooks/test/keyboard-navigation-test.js

```diff
@@ -0,0 +1,459 @@
+import { render } from 'preact';
+import { useRef } from 'preact/hooks';
+import { act } from 'preact/test-utils';
+
+import { useArrowKeyNavigation, useFocusCapture } from '../keyboard-navigation';
+import { waitFor } from '../../test-util/wait';
+
+function Toolbar({ navigationOptions = {} }) {
+  const containerRef = useRef();
+
+  useArrowKeyNavigation(containerRef, navigationOptions);
+
+  return (
+    <div ref={containerRef} data-testid="toolbar">
+      <button data-testid="bold">Bold</button>
+      <button data-testid="italic">Italic</button>
+      <button data-testid="underline">Underline</button>
+      <a href="/help" target="_blank" data-testid="help">
+        Help
+      </a>
+    </div>
+  );
+}
+describe('keyboard-navigation hooks', () => {
+  function currentItem() {
+    return document.activeElement.innerText;
+  }
+  describe('useArrowKeyNavigation', () => {
```

Comment:
There is only one small change to the actual tests for `useArrowKeyNavigation`, which I'll call out in a comment.